planner/cascades/implementation_rules.go

```diff
@@ -130,7 +133,11 @@ func (r *ImplTableGather) Match(expr *memo.GroupExpr, prop *property.PhysicalPro
 func (r *ImplTableGather) OnImplement(expr *memo.GroupExpr, reqProp *property.PhysicalProperty) (memo.Implementation, error) {
```

Comment:
Rename it to something like `TiKVSingleReadGather` in next pr?

Reply:
LGTM